aspnetcore/security/cookie-sharing.md

```diff
@@ -83,6 +105,20 @@ When using cookies directly:
 
 [!code-csharp[](cookie-sharing/sample/CookieAuth.Core/Startup.cs?name=snippet1)]
 
+Data protection keys and the app name must be shared among apps. In the sample apps, `GetKeyRingDirInfo` returns the common key storage location to the [PersistKeysToFileSystem](/dotnet/api/microsoft.aspnetcore.dataprotection.dataprotectionbuilderextensions.persistkeystofilesystem) method. Use [SetApplicationName](/dotnet/api/microsoft.aspnetcore.dataprotection.dataprotectionbuilderextensions.setapplicationname) to configure a common shared app name (`SharedCookieApp` in the sample). For more information, see [Configuring Data Protection](xref:security/data-protection/configuration/overview). 
+
+In ASP.NET Core apps, an alternative approach is not to use `PersistKeysToFileSystem` but to supply the key location to [DataProtectionProvider.Create(DirectoryInfo)](/dotnet/api/microsoft.aspnetcore.dataprotection.dataprotectionprovider.create#Microsoft_AspNetCore_DataProtection_DataProtectionProvider_Create_System_IO_DirectoryInfo_):
+
+```csharp
+services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
```

Comment:
You're using inconsistent auth schemes between samples. Use "Identity.Application" everywhere and put a note at the top that then need to match so you selected "Identity.Application" because it's the value used by ConfigureApplicationCookie.

Reply:
Do we need to mirror this example for ConfigureExternalCookie?